src/test/java/com/aws/greengrass/clientdevices/auth/certificate/BackgroundCertificateRefreshTest.java

```diff
@@ -346,8 +351,22 @@ void GIVEN_cloudFailure_WHEN_verifyingAThingCertificateAttachment_THEN_refreshTa
         assertEquals(backgroundRefresh.getLastRan(), now);
     }
 
+    @Test
+    void GIVEN_deviceBootUpAndNetworkUp_WHEN_backgroundRefreshStarts_THEN_itRunsOnlyAfterNetworkWasPreviouslyDown() {
```

Comment:
This isn't testing the behavior we want. If the network flaps right away after CDA starts, we still don't want it to refresh until the 24 hour mark, right?